src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -1499,6 +1499,7 @@ def _create_self_signed_cert(start_date, end_date):  # pylint: disable=too-many-
         with open(cert_file, "rt") as f:
             cert_string = f.read()
             cf.write(cert_string)
+    os.chmod(creds_file, 0o600)
```

Comment:
pls also add description in help file about cert file permission

Reply:
added

Reply:
How about adding some comments for readers who are not familiar with Linux permission?

Reply:
added